src/backend/parser/parse_expr.c

```diff
@@ -931,6 +931,25 @@ rewrite_scope_identity_call(ParseState *pstate, Node **lexpr, Node **rexpr)
 	if (!(*lexpr) || !(*rexpr))
 		return;
 
+	func_expr = (FuncExpr *) (*rexpr);
+	if (IsA(*rexpr, FuncExpr) &&
+		strstr(get_func_name(func_expr->funcid), "like_escape") != NULL)
```

Comment:
why strstr?

Reply:
I found strcmp() makes more sense while comparing the string, i will update this in next PR.